src/api/configuration.cpp

```diff
@@ -125,14 +125,21 @@ namespace mamba
                                     the current working directory, use './some_prefix')")
                                                 .c_str());
                 }
-                prefix = fs::absolute(prefix);
             }
             else
             {
                 bool use_fallback = config.at("use_target_prefix_fallback").value<bool>();
                 if (use_fallback)
                     prefix = std::getenv("CONDA_PREFIX") ? std::getenv("CONDA_PREFIX") : "";
             }
+
+#ifdef _WIN32
+            std::string sep = "\\";
```

Comment:
do we not have a function for this? ... 

or is there a function in fs::path?

Reply:
there is preferred_seperator in https://en.cppreference.com/w/cpp/filesystem/path

Reply:
I searched for, but no.
maybe we can create one if you prefer

Reply:
there is preferred_seperator in https://en.cppreference.com/w/cpp/filesystem/path